src/rest/connector/libs/iosxe/implementation.py

```diff
@@ -96,6 +96,7 @@ def connect(self,
         log.debug("Timeout: %s" % timeout)
         self.content_type = default_content_type
 
+        self.verify = self.connection_info.get('verify', True)
```

Comment:
shouldn't this default to False? 
```suggestion
        self.verify = self.connection_info.get('verify', False)
```

Reply:
As of today, it defaults to `True` for the `get` requests (as no `verify` parameter is passed there).
Furthermore, I guess a good security practice is, to enable SSL/TLS certificate validation by default and only disable it, if there are specific reasons.

Reply:
I don't argue, even if test tools often interact with lab devices where certificate discipline is often, err, sloppy.. 
but in the PR description you mentioned 
```
            verify: False                  # <-- Disable certificate validation (default)            
```

Reply:
Sorry @oboehmer .. my bad. I updated the PR description above